EventFilter/GEMRawToDigi/src/GEMRawToDigi.cc

```diff
@@ -3,7 +3,8 @@
  */
 #include "EventFilter/GEMRawToDigi/interface/GEMRawToDigi.h"
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
-
+// temp
+#include <iostream>
```

Comment:
this looks like a leftover from some debugging session.

Reply:
yep. will remove